tests/suites/test_suite_pkwrite.function

```diff
@@ -61,6 +61,7 @@ void pk_write_key_check(char *key_file)
     memset(buf, 0, sizeof(buf));
     memset(check_buf, 0, sizeof(check_buf));
 
+    USE_PSA_INIT();
```

Comment:
We want to add `USE_PSA_DONE();` at the end too, then (for the benefit of tools the check for resource leaks).

Reply:
Note: the missing `USE_PSA_DONE();` is actually causing all the errors like:
```
Private key write check RSA ....................................... [mutex: 2 leaked] FAILED
  Mutex usage error
  at line 217, src/threading_helpers.c
```
in `all.sh` components using a configuration based on `full` in the CI.